src/shared/models/fees/bitcoin-fees.model.ts

```diff
@@ -7,7 +7,7 @@ export interface AverageBitcoinFeeRates {
 }
 
 export const btcTxTimeMap: Record<keyof AverageBitcoinFeeRates, string> = {
-  fastestFee: '~10 – 20min',
+  fastestFee: '~10 - 20min',
```

Comment:
Please revert, this should use an en dash not a hyphen. @mica000 please make sure designs reflect this

Reply:
https://www.grammarly.com/blog/hyphens-and-dashes/

Reply:
Ok, will do, wasn't matching designs so thought we wanted changed. Will revert.